Package.swift

```diff
@@ -4,7 +4,7 @@ import PackageDescription
 let package = Package(
   name: "Pixel",  
   platforms: [
-    .iOS(.v12)
+    .iOS(.v14)
```

Comment:
Can we keep v12?

Reply:
It needs to be moved to at least iOS 13. It won't compile from SPM on iOS 12 because some of the dependencies use 13+ APIs.

Reply:
hm, okay.
I'll check it later